Frontend/ui-customer/src/Helpers/AppVersionHelper.purs

```diff
@@ -0,0 +1,95 @@
+module Helpers.AppVersionHelper where
```

Comment:
since the file is already inside Helper folder we can rename it to not include `Helper` in file Name